### ISSUE TITLE
Feature request: Ruler

### ISSUE THREAD

**Issue by U235a** [trufanov-nok/scantailor#32]
Хотелось бы прикрутить к интерфейсу виджета с изображением страницы линейки, за основу можно взять линейки из программы Scribus, исходники открыты и там тоже Qt5.
 При нажатии мышью на линейку, можно рисовать линию, которой можно проверить вертикальность или горизонтальность элементов страницы, при отпускании кнопки мыши или выходе указателя из границ линейки линия должна пропадать.

**Comment by ashipunov** [comment]
Good idea! Yes, ruler would be nice to have. Again, it could be switched on and off in settings.
